docs/HowToInstall.md

```diff
@@ -0,0 +1,99 @@
+# How to install?
+
+## Prerequisite
+
+ONE-vscode is an [extension](https://code.visualstudio.com/docs/editor/extension-marketplace) of [vscode](https://code.visualstudio.com/). To use ONE-vscode, you should install vscode first.
+
+### vscode and its extensions
+
+Let's install vscode.
+- You can download vscode [HERE](https://code.visualstudio.com/download)
+- Official document: [setup](https://code.visualstudio.com/docs/setup/setup-overview)
+- (You should also install npm for vscode)
```

Comment:
I am not sure if end-user need to install npm. Do they need npm?

Reply:
You're right. I'll remove it soon. 👍